eng/scripts/Cadl-Project-Generate.ps1

```diff
@@ -42,6 +42,17 @@ $npmWorkingDir = Resolve-Path $tempFolder/$innerFolder
 $mainCadlFile = If (Test-Path "$npmWorkingDir/client.cadl") { Resolve-Path "$npmWorkingDir/client.cadl" } Else { Resolve-Path "$npmWorkingDir/main.cadl"}
 $resolvedProjectDirectory = Resolve-Path "$ProjectDirectory/src"
 
+if (Test-Path "$npmWorkingDir/cadl-project.yaml") {
+    $cadlProjectYaml = Get-Content -Path "$npmWorkingDir/cadl-project.yaml" -Raw
+
+    Install-ModuleIfNotInstalled "powershell-yaml" "0.4.1" | Import-Module
+    $yml = ConvertFrom-YAML $cadlProjectYaml
+    if ( $yml["emitters"]["@azure-tools/cadl-csharp"]["sdk-folder"]) {
+        $yml["emitters"]["@azure-tools/cadl-csharp"]["sdk-folder"] = ""
+    }
+    (ConvertTo-Yaml $yml) | Out-File "$npmWorkingDir/cadl-project.yaml"
```

Comment:
Do we need this work at all? Or is this just temporary until the cadl team fixes their output parameter support?

Reply:
@weshaggard  Yes. it is temporary. And will be removed when cadl support output parameter.